python/ray/tests/test_output.py

```diff
@@ -46,8 +46,7 @@ def test_logger_config_with_ray_init():
     """
 
     out_str = run_string_as_driver(script)
-    assert "INFO" in out_str, out_str
```

Comment:
why do we change it? I think this shouldn't be changed?

Reply:
Ahh, yes that line can stay - the other line must change because the logger name is not included in the log message format..

Reply:
Can you just add the filename? E.g., `worker.py`